### ISSUE TITLE
Allowing the NVDA key to be any key on the keyboard

### ISSUE THREAD

**Issue by a-singer** [nvaccess/nvda#8836]
I think this might be one of the odd issues to which the template doesn't apply, it is a feature request. If I'm wrong, please let me know in closing the issue, and I will follow the template to the best of my ability. 

I am using NVDA from iOS through a VNC application. I find that this works fairly well except that the application will not properly pass through the capslock or insert key as a modifier. I am, therefore, unable to use the NVDA commands without another program being involved to modify the keyboard. I have used autohotkey to map the right alt key to the insert key. Sharpkeys and other registry-based keyboard mapping applications don’t work. I would like the ability to have the NVDA key set to any key on the keyboard that Windows can see. I have seen keyboards with, for example, the ` key next to the space and there is always, of course, one of the alt keys. Would this be wise and possible? I have searched the issue archives but have only found one similar issue, issue 225, which has the same request but as a modification for the mac. That issue, made in 2009, is still open but I’m not sure of its status. Given that NVDA, unlike Jaws, will happily work with VNC input, this may be worth doing as I have yet to see a VNC application which sends the capslock key as a modifier properly.

**Comment by ehollig** [comment]
Related issues: #225, #7995

**Comment by leonardder** [comment]
Thanks for your extensive request. I think this is actually a duplicate of #225. Though it talks about the Mac as a use case, the request is strictly spoken the same. Therefore, I'm closing this in favour of that issue.
